pkg/agent/openflow/pipeline.go

```diff
@@ -2277,6 +2278,7 @@ func (f *featureService) serviceLearnFlow(groupID binding.GroupIDType,
 	nodePortAddress bool) binding.Flow {
 	// Using unique cookie ID here to avoid learned flow cascade deletion.
 	cookieID := f.cookieAllocator.RequestWithObjectID(f.category, uint32(groupID)).Raw()
+	isIPv6 := netutils.IsIPv6(svcIP)
```

Comment:
why not use "isIPv6 := (svcIP.To4() == nil)" ?

Reply:
I think `netutils.IsIPv6(svcIP)` could be more readable.